cogs/polls.py

```diff
@@ -2,23 +2,29 @@
 from discord.ext import commands
 import random
 import json
+from typing import Final, Dict
 
-class Polls(commands.Cog):
-    
-    def __init__(self, client):    
-        self.client = client
+class PollsCog(commands.Cog):
```

Comment:
Add documentation for:
* The module
* The classes
* All functions
* All class variables
* All object variables